docs/next/util/redirectUrls.json

```diff
@@ -479,11 +479,6 @@
     "destination": "/integrations/spark",
     "statusCode": 302
   },
-  {
```

Comment:
@dpeng817 sorry, what I meant is we'll still want these redirects, but point them to the Airlift tutorial, or `dagster-airlift` integration. That way if someone lands on that page externally, they'll be routed to the new up-to-date info.

Reply:
oof I understand now

Reply:
ok NOW it's fixed lol

Reply:
Nice, thank you!